docs/PacketFence_PKI_Guide.asciidoc

```diff
@@ -161,7 +216,9 @@ Place yourself in the folder where CA was created.
 
 For the RADIUS server authentication you will have 2 files, the certificate file and private key, you need both for RADIUS to work properly.
 
-Now you can use the *send certificate* or *download certificate* to have it available. You will receive by email or in your downloads section the certificate in P12 format. You need to open it, your operating system should be able to recognize it and will ask you for a password. You have received this password by mail. After you opened it you should have 2 files in the P12 object, the certificate file and the private key, both will need to be copy into the ssl directory. 
+Now you can use the *send certificate* or *download certificate* to have it available. You will receive by email or in your downloads section the certificate in P12 format. 
```

Comment:
I'm a user and I'm now wondering where the f*ck are these buttons.

Also, case the buttons text properly : *Download certificate*